### ISSUE TITLE
Lemur 1.2.0 : Missing file-saver.js , a piece of internet removed

### ISSUE THREAD

**Issue by Dan33l** [Netflix/lemur#4176]
Hi there,
I was able to install Lemur few weeks ago. But now, the `make release` command abort with following error message :
```
bower file-saver.js#~1.20150507.2       ECMDERR Failed to execute "git ls-remote --tags --heads https://github.com/Teleborder/FileSaver.js.git", exit code of #128 remote: Support for password authentication was removed on August 13, 2021. remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication. fatal: Authentication failed for 'https://github.com/Teleborder/FileSaver.js.git/'

Additional error details:
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/Teleborder/FileSaver.js.git/'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! Lemur@ postinstall: `bower install --allow-root --config.interactive=false`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the Lemur@ postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/lemur/.npm/_logs/2022-08-29T12_36_59_986Z-debug.log

```
Without this piece, Lemur seems uninstallable.

**Comment by Dan33l** [comment]
As a temporary workaround to be able to install Lemur for my tests :
* remove line about `angular-file-saver` in file `bower.json`
* run command `npm install angular-file-saver --save-dev`
* run command `make release`

**Comment by Dan33l** [comment]
The proposed workaround looks not working.
Even if `make release` pass the WebUI does not work. In the web browser, the console gives the following error message:
```
Error: [$injector:modulerr] Failed to instantiate module lemur due to:
[$injector:modulerr] Failed to instantiate module ngFileSaver due to:
[$injector:nomod] Module 'ngFileSaver' is not available! You either misspelled the module name or forgot to load it. If registering a module ensure that you specify the dependencies as the second argument.
http://errors.angularjs.org/1.4.9/$injector/nomod?p0=ngFileSaver
minErr/<@http://127.0.0.1:8080/scripts/vendor.js:17042:12
setupModuleLoader/</module/<@http://127.0.0.1:8080/scripts/vendor.js:18980:17
ensure@http://127.0.0.1:8080/scripts/vendor.js:18904:38
module@http://127.0.0.1:8080/scripts/vendor.js:18978:14
createInjector/loadModules/<@http://127.0.0.1:8080/scripts/vendor.js:21421:22
forEach@http://127.0.0.1:8080/scripts/vendor.js:17315:20
loadModules@http://127.0.0.1:8080/scripts/vendor.js:21405:12
createInjector/loadModules/<@http://127.0.0.1:8080/scripts/vendor.js:21422:40
forEach@http://127.0.0.1:8080/scripts/vendor.js:17315:20
loadModul…
vendor.js:29696:24
```

**Comment by Dan33l** [comment]
I tested release on current master https://github.com/Netflix/lemur/commit/1317a984225f18b13bb990c27d7d9df4c65eb8e0.

I was able to run `make release` without trouble. So probably a fix was done since Lemur release 1.2.

Is it planed to produce a new release for Lemur ?

**Comment by hosseinsh** [comment]
we just did a release:
https://github.com/Netflix/lemur/releases/tag/v1.3.0
